ui/public/config.json

```diff
@@ -18,6 +18,14 @@
     "404": "assets/404.png",
     "500": "assets/500.png"
   },
+  "externalLinksIcon": "",
+  "externalLinks": [
```

Comment:
`plugins` feature in config.json does exactly the same I think cc @utchoang can you review and check this?